.github/actions/snippet-tests/action.yml

```diff
@@ -0,0 +1,18 @@
+name: Documentation Snippets Tests
+description: Run Documentation Snippets Tests
+inputs:
+  CYPRESS_RECORD_KEY_DOC:
+    description: Cypress project key
+    required: true
+runs:
+  using: "composite"
+  steps:
+    - name: Build the Stack
+      run: |
+        npm install
+        npm run build
```

Comment:
I would really prefer to build TS here -> https://github.com/kuzzleio/sdk-javascript/blob/master/.ci/test-docs.sh#L7
Reason:
When you run your test locally with the provided script `npm run doc-testing` TS is not build, because it launches directly the `test-docs.sh`, and It would be nice to have it automatic